### DISCUSSION TITLE
docker版alist无法读取/home文件夹下的文件

### DISCUSSION THREAD

**Discussion by snowy-yang** [AlistGo/alist#6041] [Q&A] [answered]
docker版alist无法读取/home文件夹下的文件

**Comment by darkelfzdy** [comment]
建docker容器的时候需要映射才能读取，加一条：-v /home:/home

**Reply by snowy-yang** [reply]
请问其他目录也是吗

**Reply by snowy-yang** [reply]
docker run -d --restart=unless-stopped -v /etc/alist:/opt/alist/data -p 5244:5244 -e PUID=0 -e PGID=0 -e UMASK=022 --name="alist" xhofe/alist:latest 怎么添加这条命令

**Reply by darkelfzdy** [reply, marked as answer]
docker run -d --restart=unless-stopped -v /etc/alist:/opt/alist/data -v /home:/home -p 5244:5244 -e PUID=0 -e PGID=0 -e UMASK=022 --name="alist" xhofe/alist:latest 
如果想对本地哪个目录，就想这样加进去，这就是映射。因为docker是封闭的容器，如果不映射是没法读取本地内容的

**Reply by snowy-yang** [reply]
这可太草了

**Comment by snowy-yang** [comment]
啊，还有这种事

**Reply by darkelfzdy** [reply]
所以alist的docker有时候折腾会比较麻烦，可以看看我之前有个离线下载问题，把我折腾惨了
#5907
